.github/workflows/reusable-macos.yml

```
@@ -27,6 +27,13 @@ jobs:
       fail-fast: false
       matrix:
         os: ${{fromJson(inputs.os-matrix)}}
+        isFork:
```

We tend to use kebab-case elsewhere:
```suggestion
        is-fork:
```